Docs: sync visualization docs with source (pt2) (#126813)
grafana/grafana: docs/sources/visualizations/panels-visualizations/visualizations/flame-graph/index.md

Required fields:

| Field name | Type           | Description                                                                                                                 |
| ---------- | -------------- | --------------------------------------------------------------------------------------------------------------------------- |

> | level      | number         | The nesting level of the item. In other words how many items are between this item and the top item of the flame graph.     |

Reviewer: ```suggestion
| level      | number         | The nesting level of the item, which represents how many items are between this item and the top item of the flame graph.     |
```

## Top table mode

The top table shows the functions from the profile in table format. The table has three columns: **Symbol**, **Self**, and **Total**. The table is sorted by self time by default, but can be reordered by total time or symbol name by clicking the column headers. Each row represents aggregated values for the given function if the function appears in multiple places in the profile.

> {{< figure src="/media/docs/grafana/panels-visualizations/screenshot-flamegraph-toptable-v12.0.png" max-width="700px" alt="Table view">}}

Author: False positive. This is inside a file path
